packages/table/src/table.tsx

```diff
@@ -660,7 +666,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     public componentDidMount() {
         this.validateGrid();
 
-        this.locator = new Locator(this.mainQuadrantElement, this.scrollContainerElement, this.cellContainerElement);
+        this.locator = new Locator(this.rootTableElement, this.scrollContainerElement, this.cellContainerElement);
```

Comment:
oh man, are you sure this is right? changing the element of the locator makes me nervous